sancus/stubs/multiplication-stub/Makefile

```diff
@@ -0,0 +1,26 @@
+include ../Makefile.include
```

Comment:
you probably want to automatically run your post-processing Python script here too after executing the attack (on either FPGA or simulator) -- e.g., have a look at sancus-examples/sensor-reader I believe